xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+      </para>
+      <procedure>
+         <remark>toms 2020-05-11: Is step 1 and step 2 really needed? Can I just
+         jump to step 3?</remark>
+         <step>
+            <para>
+               On &node1;, initialize your cluster:
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster init -y</screen>
+         </step>
+         <step>
+            <para>
+               On &node2;, join the cluster:
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster join -c &node1; -y</screen>
+         </step>
+         <step>
+            <para>
+               On &node1; and &node2;, bootstrap the <literal>qdevice</literal> stage.
+               Usually, in most cases the default settings are fine.
+               Provide at least <option>--qnetd-hostname</option> and the hostname or
+               IP address of the &qnet; server (&node3; in our case):
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster init qdevice --qnetd-hostname=&node3;</screen>
+            <para>
+               If you want to change the default settings, get a list of all
+               possible options with the command <command
+                  >crm cluster init qdevice --help</command>.
+               All options related to &qdevice; start with
+               <option>--qdevice-<replaceable>NAME</replaceable></option>.
+            </para>
+         </step>
+      </procedure>
+      <para>
+         If you have used the default settings, it will create a &qdevice; with TLS enabled
+         and using the FFSplit algorithm.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-heuristic">
+      <title>Setting up a &qdevice; with Heuristics</title>
+      <para>
+         If you need finer control over how votes are determined, use heuristics.
+         Heuristics are a set of commands which are executed in parallel.
+      </para>
+      <para>
+         For this purpose, the command <command>crm cluster init qevice</command>
+         provides the option <option>--qdevice-heuristics</option>. You can
+         pass one or more commands (separated by semicolon) with absolute paths.
+      </para>
+      <para>
+         For example, if your own command for quorum calculation is located at
+         <filename>/usr/sbin/my-script.sh</filename> you could run it on
+         one of your cluster nodes as follows:
+      </para>
+      <screen>&prompt.root;<command>crm</command
+> cluster init qdevice --qdevice-hostname=&node3; \
+     --qdevice-heuristics=/usr/sbin/my-script.sh</screen>
+      <para>
+         The command can be anything (Shell, Python, Ruby etc). It has to signal the result
+         by returning an exit code. The script returns zero as exit code if the heuristic is
+         correct. If not, the script needs to signal that with an exit code like one (not
+         equal to zero).
```

Comment:
I would rephrase it in somewhat easier like "The command can written in any language such as Shell, Python, Ruby etc. It returns 0 if succeded, otherwise it returns an error code."

Reply:
Wonderful, I'll take that from Alexei. :+1: